projects/commudle-admin/src/app/feature-modules/community-builds/components/community-build-h-list-item/community-build-h-list-item.component.html

```diff
@@ -18,32 +18,39 @@
           <h3 class="name">
             {{ communityBuild.name }}
           </h3>
-          <p class="text-info show-details">
-            <nb-icon icon="link" status="info"></nb-icon> Show Details
+          <p class="text-info show-details"  >
+            <!-- <nb-icon icon="link" status="info"></nb-icon> Show Details -->
+            {{ buildDescription }};
+          </p>
+
+          <div class="author-stats">
+            <div class="author">
+              <app-user-profile-horizontal [user]="communityBuild.user" [socialMediaLinks]="false"></app-user-profile-horizontal>
+            </div>
+
+        <div class="stats">
+          <p class="stars">
+            <app-votes-display
+              [votableType]="'CommunityBuild'"
+              [votableId]="communityBuild.id"
+              [icon]="'star'"
+              >
+            </app-votes-display>
+          </p>
+          <p class="comments" [routerLink]="['/builds', communityBuild.slug]">
+            <span *ngIf="communityBuild.chat_messages_count > 0">
+              {{ communityBuild.chat_messages_count }}
+            </span>
+            <nb-icon icon="message-circle-outline"></nb-icon>
           </p>
-          <div class="author">
-            <app-user-profile-horizontal [user]="communityBuild.user" [socialMediaLinks]="false"></app-user-profile-horizontal>
-          </div>
         </div>
-        <div class="image" *ngIf="communityBuild.images.length > 0">
-          <img src="{{ communityBuild.images[0].url }}" alt="{{ communityBuild.name }}">
         </div>
+
+      </div>
+      <div class="image" *ngIf="communityBuild.images.length > 0">
+        <img src="{{ communityBuild.images[0].url }}" alt="{{ communityBuild.name }}">
```

Comment:
The ngIf can be moved to img and background can be added to this div itself in case of no image using ngclass